tests/output/LoggingTest.py

```diff
@@ -30,3 +31,17 @@ def test_json_logging(self):
         self.assertEqual(logs_list[2]['level'], 'WARNING')
         self.assertEqual(logs_list[3]['message'], 'This is error log.')
         self.assertEqual(logs_list[3]['level'], 'ERROR')
+
+    def test_file_logging(self):
+        configure_logging()
+        debug_text = 'This is debug log.'
+        logging.debug(debug_text)
+        uut = open('.coala.log', 'r')
+        self.assertIn(debug_text, uut.read())
+        uut.close()
```

Comment:
use context managers, then you don't have to close yourself:
```
with open(...) as fl:
    self.assertIn(debug_test, fl.read())
```
and also don't call the file `uut`, this is normally a component of the program currently tested. So a file would be called something like `fut` or so (file-under-test, though that sounds crappy :3)^^

btw: `r` is default, so you don't need to provide that explicitly ;D